examples/restaurant-approval/nextjs-ui/app/globals.css

```
@@ -0,0 +1,256 @@
+:root {
+  --bg: #f6efe4;
+  --paper: rgb(255 249 240 / 0.92);
```

Stylelint (17.12.0)

[error] Expected "0.92" to be "92%" (alpha-value-notation)

(alpha-value-notation)